WORKSPACE

```diff
@@ -107,12 +128,12 @@ load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
 
 aspect_bazel_lib_dependencies()
 
-BAZEL_INSTALLER_VERSION_LINUX_X86_64_SHA = "c0161a346b9c0d00e6eb3d3e8f9c4dece32f6292520248c5ab2e3527265601c1"
+BAZEL_INSTALLER_VERSION_LINUX_X86_64_SHA = "7e749f59fa4c4430fbc49ea38b0bc0a41dd1f460e8671c90978d70c2166bf0c4"
```

Comment:
This is the same SHA as the one you proposed in https://github.com/sorbet/sorbet/pull/9602/files#diff-64b1cd877dd573d06363688eb14589e697e6b4ca3a1e51fd0d88120050e1e896. But the CI there was failing. Are you sure this is a correct SHA?

Reply:
If you look at the latest SHA added in https://github.com/sorbet/sorbet/pull/9602/commits/5464e30159e02f7eb0e46c1fa3b596d062492e7b, that is indeed `7e749f59fa4c4430fbc49ea38b0bc0a41dd1f460e8671c90978d70c2166bf0c4` for x86_64 Linux.